dll/cpl/powercfg/lang/ro-RO.rc

```diff
@@ -4,110 +4,110 @@
     CONTROL "", IDC_ALARMBAR1, "msctls_trackbar32", TBS_HORZ | TBS_AUTOTICKS | WS_TABSTOP, 47, 32, 194, 15
-    PUSHBUTTON "A&cțiune alarmă…", -1, 148, 50, 88, 14, WS_DISABLED
+    PUSHBUTTON "A&cţiune de alertare…", -1, 148, 50, 88, 14, WS_DISABLED
     LTEXT "Notificare:", -1, 28, 67, 89, 10
     LTEXT "?", IDC_ALARMMSG1, 118, 67, 125, 10
     LTEXT "Acțiune:", -1, 28, 79, 89, 10
     LTEXT "?", IDC_ALARMAKTION1, 118, 79, 125, 10
-    LTEXT "Exec. program:", -1, 28, 91, 89, 10
+    LTEXT "Executare program:", -1, 28, 91, 89, 10
     LTEXT "?", IDC_ALARMPROG1, 118, 91, 125, 10
-    GROUPBOX "Alarmă baterie descărcată", -1, 6, 115, 239, 99
-    AUTOCHECKBOX "Alarmează „baterie d&escărcată” când va ajunge la:", IDC_ALARM2, 15, 125, 225, 10, WS_TABSTOP
+    GROUPBOX "Alertă baterie critică", -1, 6, 115, 239, 99
+    AUTOCHECKBOX "Acti&vează alerta pentru baterie critică, când nivelul de încărcare atinge:", IDC_ALARM2, 15, 125, 225, 10, WS_TABSTOP
     LTEXT "xxx%", IDC_ALARMVALUE2, 28, 140, 20, 10
     CONTROL "", IDC_ALARMBAR2, "msctls_trackbar32", TBS_HORZ | TBS_AUTOTICKS | WS_TABSTOP, 47, 140, 194, 15
-    PUSHBUTTON "Acți&une alarmă…", -1, 148, 158, 88, 14, WS_DISABLED
+    PUSHBUTTON "&Acţiune de alertare…", -1, 148, 158, 88, 14, WS_DISABLED
     LTEXT "Notificare:", -1, 28, 175, 89, 10
     LTEXT "?", IDC_ALARMMSG2, 118, 175, 125, 10
     LTEXT "Acțiune:", -1, 28, 187, 89, 10
     LTEXT "?", IDC_ALARMAKTION2, 118, 187, 125, 10
-    LTEXT "Exec. program:", -1, 28, 199, 89, 10
+    LTEXT "Executare program:", -1, 28, 199, 89, 10
     LTEXT "?", IDC_ALARMPROG2, 118, 199, 125, 10
 END
 
 IDD_PROPPAGEADVANCED DIALOGEX 0, 0, 252, 220
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Avansate"
+CAPTION "Complex"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_ACDC, IDI_ACDC, 7, 7, 0, 0
-    LTEXT "Specificați opțiunile de economisire ale consumului de energie al calculatorului.", -1, 37, 6, 208, 20
+    LTEXT "Selectaţi setările pentru economisirea energiei.", -1, 37, 6, 208, 20
     GROUPBOX "Opțiuni", -1, 6, 30, 240, 65
-    AUTOCHECKBOX "Afișează pictogramă în &zona de notificare", IDC_SYSTRAYBATTERYMETER, 17, 43, 220, 10, WS_TABSTOP
-    AUTOCHECKBOX "Solicită &reautentificare la revenirea din starea de veghe", IDC_PASSWORDLOGON, 17, 60, 220, 10, WS_TABSTOP
-    AUTOCHECKBOX "Înt&unecă ecranul când calculatorul merge pe baterii", IDC_VIDEODIMDISPLAY, 17, 78, 220, 10, WS_TABSTOP
-    GROUPBOX "Consecințe ale acțiunilor de utilizator", -1, 6, 101, 240, 110
-    LTEXT "&Plierea calculatorul portabil determină:", IDC_SLIDCLOSE, 15, 115, 222, 10
+    AUTOCHECKBOX "Afişa&re pictograme pe bara de activităţi", IDC_SYSTRAYBATTERYMETER, 17, 43, 220, 10, WS_TABSTOP
+    AUTOCHECKBOX "&Solicitare parolă când computerul iese din starea de veghe", IDC_PASSWORDLOGON, 17, 60, 220, 10, WS_TABSTOP
+    AUTOCHECKBOX "&Umbrire afişare când se alimentează de la baterii.", IDC_VIDEODIMDISPLAY, 17, 78, 220, 10, WS_TABSTOP
```

Comment:
```suggestion
    AUTOCHECKBOX "&Umbrire afişare când se alimentează de la baterii", IDC_VIDEODIMDISPLAY, 17, 78, 220, 10, WS_TABSTOP
```
MS does (almost?) never end texts in AUTOCHECKBOXES with a dot. I am pretty sure that the dot in the end is wrong.

Reply:
Resolved, thanks.